GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -526,7 +526,11 @@ private void TxtBranchCriterion_KeyDown(object sender, KeyEventArgs e)
 
         private void OnPreviewKeyDown(object sender, PreviewKeyDownEventArgs e)
         {
-            if (e.KeyCode == Keys.F3)
+            if (e.KeyCode == Keys.F2)
```

Comment:
Can we use a `switch` statement here?

Reply:
Yes, I think it makes sense, especially if the list of hotkeys handled tends to grow,